test/incident-search-api-test.js

```diff
@@ -41,7 +41,7 @@ describe('Incident Search API', () => {
       var priority = '3'
       this.request.get('/api/v1/incidents?priority=' + priority, (error, response) => {
         if (error) { done(error); }
-        var parsed = JSON.parse(response.body);
+        var parsed = JSON.parse(response.body).incidents;
```

Comment:
Could probably have a tiny helper here to save you from having to add `.incidents` everywhere.